asset/dcr/addresses.go

```diff
@@ -10,68 +10,81 @@ import (
 )
 
 // AddressesByAccount handles a getaddressesbyaccount request by returning
-// all addresses for an account, or an error if the requested account does
-// not exist. Does not include the current address that can be retrieved with
-// w.mainWallet.CurrentAddress.
-func (w *Wallet) AddressesByAccount(ctx context.Context, account string) ([]string, error) {
+// external addresses for an account, or an error if the requested account does
+// not exist. Returns used and unused addresses up to n. No unused addresses
+// are returned if n is zero. All used addresses are returned if that n is zero.
```

Comment:
"n" is not used anywhere as a parameter.

Reply:
Using the correct names now https://github.com/decred/libwallet/compare/928296b2d0686c56a002f5a1ba1ebfd75f72c029..570e70e4cfa1f4455248262eeb560ca15c516de8